events/app/views/pages/event.njk

```diff
@@ -16,65 +16,94 @@
 <script type="application/ld+json">
   {{ event | jsonLd('eventLd') | safe }}
 </script>
+<div class="row">
+  <div class="container {{ {s:2} | spacingClasses({margin: ['top']}) }}">
+    {% componentV2 'tags', tags %}
+  </div>
+</div>
 
-<article class="container">
-  {# Event title #}
-  <p>{{ event.title }}</p>
-
-  {# Tags #}
-  {% componentV2 'tags', tags %}
-
-  {# Format #}
-  <p>{{ event.format.title }}</p>
-
-  {# Access information #}
-  {% if (event.accessOptions.size !== 0) %}
-    {% for accessOption in event.accessOptions %}
-      {% icon 'a11y/' + (accessOption.accessOption.acronym | lower), accessOption.accessOption.title %}
-    {% endfor %}
-  {% endif %}
+<div class="row">
+  <div>
+    {% if event.promo.image %}
+      {% componentV2 'image', event.promo.image, [], { lazyload: true } %}
```

Comment:
I suppose it doesn't matter here, but I think `componentV2` expects the modifiers to be an object rather than an array.

Reply:
Lovely "leteverythingthrough" nunjucks.